src/ui/osx/TogglDesktop/Feature/TimeEntryEditor/Project/ProjectCreationView.swift

```diff
@@ -305,13 +305,15 @@ extension ProjectCreationView {
         addBtn.isEnabled = true
     }
 
-    fileprivate func selecteFirstWorkspace() {
-        guard !workspaceDatasource.items.isEmpty else { return }
-        workspaceDatasource.selectRow(at: 0)
+    fileprivate func selectDefaultWorkspace() {
+        guard let workspaceID = selectedTimeEntry?.workspaceID else { return }
```

Comment:
It's actually a better solution than fetching the default workspace in user from the Library (according to my last implementation in https://github.com/toggl-open-source/toggldesktop/pull/3844)
By doing it, we can make sure that the workspace is the same workspace with the selected Time Entry. 

Good job. Thank for the suggestion 🙌